### DISCUSSION TITLE
[Proposal] Merge GFM for wikis and repositories together.

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#62063] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Looking at the GitHub Flavourited Markdown (GFM) used for Wikis, there seems to only be one difference, which is the support of `[[page]]` and `[[title|page]]` links.

This is in my opinion a relative small difference, which could also be integrated into repositories themself, to allow a much easier linking of Markdown files within the repository.
Looking at it right now does this separation just cause annoyance towards the community as features available in repositories may not be available in wikis yet, such as the custom Note, warning and important boxes GitHub works on.
In addition can I imagine that having a separate set of GFM also adds extra work load onto the people responsible for managing and updating it.

Having both merged together into a single set would reduce the work and make updates across these parts of GitHub significantly easier.

I really hope this is considered. If it cannot be done, please give me an explanation as to why, because from what I get is there barely any reason why it should not work like this...

**Comment by sinsukehlab** [comment]
The converter used in GitHub Wiki isn't even GitHub Flavored Markdown. It's [GitHub Markup](/github/markup).
The support of [[page]] and [[title|page]] most likely comes from Wiki Markup.
